TestApp/src/main/res/AndroidManifest.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.tekchand.testapp">
```

Comment:
This file is inside the res folder. I see there is already a manifest in main. Is this being used anywhere?